From 2578eeae4c159be4430cc9e8413319ec8ada1ebd Mon Sep 17 00:00:00 2001
From: Marek 'marx' Grac <mgrac@redhat.com>
Date: Thu, 19 Mar 2015 21:15:37 +0100
Subject: [PATCH 3/4] fence_bladecenter: Fix login process

fence_bladecenter is specific because motd contains string "Last login:" which
can be easily mishandled with "login:". This dual checking is included to properly
work with CR/LF autodetection.
---
 fence/agents/bladecenter/fence_bladecenter.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fence/agents/bladecenter/fence_bladecenter.py b/fence/agents/bladecenter/fence_bladecenter.py
index 38604d5..d72c07f 100644
--- a/fence/agents/bladecenter/fence_bladecenter.py
+++ b/fence/agents/bladecenter/fence_bladecenter.py
@@ -102,7 +102,7 @@ and uses the command line interface to power on and off blades."
 	##
 	## Operate the fencing device
 	######
-	conn = fence_login(options)
+	conn = fence_login(options, "(username\s*:\s*)")
 	result = fence_action(conn, options, set_power_status, get_power_status, get_blades_list)
 	fence_logout(conn, "exit")
 	sys.exit(result)
-- 
1.9.3

